src/main/java/seedu/address/model/student/Student.java

```diff
@@ -110,9 +111,9 @@ public void addSession(Session session) {
      * Sets the list of sessions
```

Comment:
Actually the method name of "AddSessions" and the description that we "Set" the session doesn't really match. Maybe changing the description to "Adds any number of sessions to the current student's session list." or something might be better.

Reply:
Nice catch didn't notice this! Will make changes, appreciate it!